src/cpp/_pythia8.cpp

```diff
@@ -244,11 +245,18 @@ PYBIND11_MODULE(_pythia8, m)
 
         ;
 
+    py::class_<SubCollisionModel, std::shared_ptr<SubCollisionModel>>(m, "SubCollisionModel")
+        // TODO add getParm
```

Comment:
Which function exactly do you want to implement?

@jncots @HDembinski , this could be a good chance to also satisfy this feature request #187 here as well?

Reply:
I would do this in a separate PR, but yeah, it is probably not complicated to implement access to the weights in a similar way.

I don't like that an info object has to be created each time when these are accessed, this seems wasteful and unnecessary. There may be a way around that if we access private members. I would like to take a look before we implement something more straight-forward but inefficient.